docs/creating_integration_manifest.md

```diff
@@ -414,9 +414,7 @@ For example:
 
 The [Integration Quality Scale](https://www.home-assistant.io/docs/quality_scale/) scores an integration on the code quality and user experience. Each level of the quality scale consists of a list of requirements. If an integration matches all requirements, it's considered to have reached that level.
 
-When your integration has no score, then don't add it to the manifest of your integration. However, be sure to look at the [Integration Quality Scale](https://www.home-assistant.io/docs/quality_scale/) list of requirements. It helps to improve the code and user experience tremendously.
-
-We highly recommend getting your integration scored.
+New integrations are required to fulfill at least the bronze tier, but we highly recommend aiming for higher tiers. Be sure to look at the [Integration Quality Scale](https://www.home-assistant.io/docs/quality_scale/) list of requirements. It helps to improve the code and user experience tremendously.
```

Comment:
Hmm, yes we highly recommend it, but I don't think that should come after the requirement for new integrations. (as currently it reads as is you can also directly commit a platinum-worthy integration to core, which is not the case, as it will most likely have too much for an initial PR like services, reauth, reconfigure and diagnostics)